### ISSUE TITLE
Saved Schematic name cannot have slashes in

### ISSUE THREAD

**Issue by OscarVanL** [maruohon/litematica#127]
When saving a Schematic with a slash in its name it fails to save without a descriptive error message.

It would be nice if there were a tooltip indicating which characters are not permitted if an invalid one is entered, or if the error was more descriptive.

![image](https://user-images.githubusercontent.com/7999692/77790144-2f3c3b00-705c-11ea-96e0-a6d2d198df6e.png)

**Comment by maruohon** [comment]
Yes I should definitely add a warning message about illegal characters, and also validate the name and print the message in that GUI before allowing it to close, before actually creating the saving task.

**Comment by Eiim** [comment]
The current situation seems to be worse (at least on Windows): Litematica reports that the schematic was saved successfully, but nothing is saved.
